test/api/timeoutforceddone.js

```diff
@@ -101,7 +105,57 @@ describe('Provider forced done by timeout', function() {
 
     var auctionProvider = pf.setAuctionProvider({
       name: 'auctionProvider',
-      libUri: '../test/fixture/lib.js',
+      libUri: 'fixture/lib.js',
+      init: function(targeting, done) {
+        setTimeout(function() {
```

Comment:
Tests such as these and several others that use `setTimeout()` introduce potential unpredictability to the test suites.

In future tests, we should address this by either:

1. Using mocks and/or fake timer libraries such as those available in http://sinonjs.org/; **or**
2. Using the pubfood library call and event history to validate expected call sequence and/or function arguments.